roles/openshift_node/tasks/storage_plugins/iscsi.yml

```diff
@@ -22,13 +22,42 @@
     - rpcbind
     - iscsid
 
+# Copy the conf file only if not exists
 - name: Template multipath configuration
   template:
     dest: "/etc/multipath.conf"
     src: multipath.conf.j2
     backup: true
+    force: no
   when: not openshift_is_atomic | bool
 
+# If the conf file already exists, update it
+- name: "Add LIO-ORG in MultiPath configuration"
+  blockinfile:
+    path: /etc/multipath.conf
+    marker: "# <!-- {mark} ANSIBLE MANAGED BLOCK -->"
+    block: |
+      devices {
```

Comment:
What if the pre-existing multipath.conf has a devices section already?

Reply:
With previously existing "devices { device}" and adding a new device as mentioned here (with duplicate device section) - it displays all the devices properly in "multipath -t"  properly coalesced - But there is still a warning at the top saying "/etc/multipath.conf line 50, duplicate keyword: devices". I will address this.

Reply:
Done